deploy/runtime/defaults/PV Battery_Third Party.txt

```diff
@@ -821,6 +821,12 @@ batt_dispatch_excl
 1
 1
 3.33506e-317
+batt_dispatch_wf_forecast_choice
+1
+1
+1
+1
+3.33505e-317
```

Comment:
#642

Reply:
@brtietz Maybe try writing 0.0 as the default to get a true 0. I've had some success with that in the past.